VideoApi/VideoApi.Domain/ParticipantBase.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace VideoApi.Domain
 {
-    public abstract class ParticipantBase : Entity<Guid>
+    public class ParticipantBase : Entity<Guid>
```

Comment:
The abstract modifier should not be removed. I think you are instantiating it in a test which is why you removed it from here. In that case, in your test you should new up the concrete class and not the base class